抢原石报错

![截图2046](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/83526546/5fa8a65a-73bd-434e-b64b-f0088b43af42)

05-31 19:00:04 [ERROR] mys_goods_tool.api | 米游币商品兑换 - 执行兑换: 用户 7*******2 商品 2023051912577 请求失败
Traceback (most recent call last):

  File "threading.py", line 995, in _bootstrap

  File "threading.py", line 1038, in _bootstrap_inner

  File "threading.py", line 975, in run

  File "concurrent\futures\thread.py", line 83, in _worker

  File "concurrent\futures\thread.py", line 58, in run

  File "apscheduler\executors\base.py", line 125, in run_job

  File "mys_goods_tool\exchange_mode.py", line 101, in exchange_begin

> File "mys_goods_tool\api.py", line 1266, in good_exchange_sync

  File "httpx\_models.py", line 756, in json

  File "json\__init__.py", line 346, in loads

  File "json\decoder.py", line 337, in decode

  File "json\decoder.py", line 355, in raw_decode

json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
05-31 19:00:04 [ERROR] mys_goods_tool.api | 米游币商品兑换 - 执行兑换: 用户 7******2 商品 2023051912577 请求失败
Traceback (most recent call last):

  File "threading.py", line 995, in _bootstrap

  File "threading.py", line 1038, in _bootstrap_inner

  File "threading.py", line 975, in run

  File "concurrent\futures\thread.py", line 83, in _worker

  File "concurrent\futures\thread.py", line 58, in run

  File "apscheduler\executors\base.py", line 125, in run_job

  File "mys_goods_tool\exchange_mode.py", line 101, in exchange_begin

> File "mys_goods_tool\api.py", line 1266, in good_exchange_sync

  File "httpx\_models.py", line 756, in json

  File "json\__init__.py", line 346, in loads

  File "json\decoder.py", line 337, in decode

  File "json\decoder.py", line 355, in raw_decode

json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
05-31 19:00:04 [ERROR] mys_goods_tool.exchange_mode | 接收兑换结果失败
Traceback (most recent call last):

  File "threading.py", line 995, in _bootstrap

  File "threading.py", line 1038, in _bootstrap_inner

  File "threading.py", line 975, in run

  File "concurrent\futures\thread.py", line 83, in _worker

  File "concurrent\futures\thread.py", line 64, in run

  File "concurrent\futures\_base.py", line 550, in set_result

  File "concurrent\futures\_base.py", line 340, in _invoke_callbacks

  File "apscheduler\executors\pool.py", line 25, in callback

  File "apscheduler\executors\base.py", line 90, in _run_job_success

  File "apscheduler\schedulers\base.py", line 835, in _dispatch_event

> File "mys_goods_tool\exchange_mode.py", line 304, in on_executed

StopIteration

--- a/mys_goods_tool/api.py
+++ b/mys_goods_tool/api.py
@@ -197,7 +197,7 @@ HEADERS_EXCHANGE = {
     "Connection":
         "keep-alive",
     "Content-Type":
-        "application/json",
+        "application/json;charset=utf-8",
     "Host":
         "api-takumi.miyoushe.com",
     "Origin":

